site/src/components/LicenseCard/LicenseCard.tsx

```diff
@@ -82,7 +85,7 @@ export const LicenseCard = ({
             width="134px" // standardize width of date column
           >
             {compareAsc(
-              new Date(license.claims.license_expires * 1000),
+              new Date((license.claims.license_expires as number) * 1000),
```

Comment:
Can we make the conversion to number in a single place above?

Reply:
If the license_expires is not always a number I'm wondering if we should care about the other types it can be 🤔. Do you know what is the case when it is not a number?

Reply:
type enforced to be a `number`